src/rprblender/properties/render.py

```diff
@@ -580,6 +580,12 @@ def init_rpr_context(self, rpr_context, is_final_engine=True, use_gl_interop=Fal
             if float(mac_vers_major) >= 14:
                 rpr_context.set_parameter(pyrpr.CONTEXT_METAL_PERFORMANCE_SHADER, 1)
 
+            # set ocio config file to blender included one
+            # TODO can use blender render set render space, and check for custom ocio setting
+            rpr_context.set_parameter(pyrpr.CONTEXT_OCIO_CONFIG_PATH, 
```

Comment:
Why do we set it under metal_enabled (in MacOS) option? Is this relates only to MacOS?
Also for RPR1 it produces raises ERROR_UNSUPPORTED.

Reply:
This was a copy and paste error from another branch.  Will fix.